build.js

```diff
@@ -0,0 +1,57 @@
+// Dependencies
+var GitHub = require("gh.js")
+  , SameTime = require("same-time")
+  , Logger = require("bug-killer")
+  , Fs = require("fs")
+  ;
```

Comment:
@IonicaBizau maybe a noob question but where are these dependencies pulling in from?